src/tools/tests/manifest2proto-test.ts

```diff
@@ -777,7 +786,25 @@ describe('manifest2proto', () => {
 
     assert.deepStrictEqual(varInput, varOutput);
     assert.deepStrictEqual(varInput.name, 'a');
-    assert.isUndefined(varInput.constraint);
+    assert.isFalse(varInput.constraint.maxAccess);
+    assert.isFalse(varOutput.constraint.maxAccess);
+  });
+
+  it('encodes max variable type for particle specs - unconstrained', async () => {
+    const manifest = await Manifest.parse(`
+    particle P
+      input: reads ~a with {*}
```

Comment:
Can you also add an example with no constraints:

```
particle P
  input: reads ~a
  output: writes ~a
```

Reply:
This test already exists (it may not show up on the diff).